### DISCUSSION TITLE
mtexColorbar issue

### DISCUSSION THREAD

**Discussion by King-wang-H** [mtex-toolbox/mtex#1101] [Ask Anything]
Hi MTEX users,
I would like to ask some questions about mtexColorbar.I want to set the specific display range of the ruler in the picture below. For example, from the original 0-1 range to 0-0.5 range, how to achieve [this?](![1](https://user-images.githubusercontent.com/85743677/122636810-c5ef8e80-d11d-11eb-8ce9-4c6a8a86829f.png))
Attached is the code in my [MTEX.](![2](https://user-images.githubusercontent.com/85743677/122636818-cd169c80-d11d-11eb-9748-dbf72e327233.png))
I sincerely hope that someone can help me to answer this question or provide relevant code information about mtexColorbar.Thank you in advance.

**Comment by kilir** [comment]
Hi,
Have a look at `CLim`.
Cheers, Rüdiger

**Reply by King-wang-H** [reply]
Thank you very much. My problem has been solved!
